broker/src/main/java/io/camunda/zeebe/broker/bootstrap/ClusterTopologyManagerStep.java

```diff
@@ -35,6 +35,16 @@ public ActorFuture<BrokerStartupContext> startup(
             (ignore, error) -> {
               if (error == null) {
                 brokerStartupContext.setClusterTopology(clusterTopologyService);
+                final var brokerInfo = brokerStartupContext.getBrokerInfo();
+                final var clusterConfiguration =
+                    brokerStartupContext.getBrokerClient().getTopologyManager().getTopology();
+                brokerInfo
+                    .setClusterSize(clusterConfiguration.getClusterSize())
+                    .setPartitionsCount(clusterConfiguration.getPartitionsCount());
+                if (clusterConfiguration.getReplicationFactor() > 0) {
+                  brokerInfo.setReplicationFactor(clusterConfiguration.getReplicationFactor());
+                }
```

Comment:
:question: Why would the replication factor be zero here? As far as I know, this should always be set.

Reply:
Not sure, I was a bit suprised as well,  I'll try to investigate as soon as I can..